.github/workflows/github-page-publish.yml

```diff
@@ -1,20 +1,24 @@
 name: Github Page Publish
 on:
   push:
-    branches: [ main ]
+    branches:
+      - 'main'
+    paths-ignore:
+      - 'test/**'
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout 🛎️
+      - name: Checkout
```

Comment:
I liked the emoji because it was highly readable, are you going to delete it?

Reply:
Ah Then I'm going to revert the emoji 😄